app/assets/src/components/views/SampleView/SampleView.jsx

```diff
@@ -296,6 +296,9 @@ class SampleView extends React.Component {
         return (
           <div className={cs.failedContainer}>
             <h6 className={cs.failed}>Sample run failed</h6>
+            <h6 className={cs.failed}>
+              {this.props.pipelineRun.error_message}
```

Comment:
Design-wise the error message could have a separate style to be more friendly (just normalcase, non-red)

Reply:
true